python/shotgun_globals/cached_schema.py

```diff
@@ -191,6 +191,9 @@ def _load_cached_status(self, project_id=None):
         if os.path.exists(status_cache_path):
             try:
                 self._bundle.log_debug("Loading cached status from '%s'" % status_cache_path)
+                # Reset the file modification time to prevent it to be removed
```

Comment:
i think we should just clean up thumbnails and caches (which are an ever growing number of items) - the schema is just a constant number of files, so no need to clean up i think.